platform/broadcom/sonic-platform-modules-dell/debian/rules

```diff
@@ -37,7 +38,8 @@ override_dh_auto_build:
 		fi; \
 		echo "making man page alias $$mod -> $$mod APIs";\
 		make -C $(KERNEL_SRC)/build M=$(MOD_SRC_DIR)/$${mod}/modules; \
-	done)
+	done); \
+	set +e;
```

Comment:
what about failure in the install phase?

Reply:
Updated.